internal/presentation/cli.go

```diff
@@ -0,0 +1,115 @@
+package presentation
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/marmelab/labyrinth/internal/model"
+)
+
+// This is the shape parts for line assembly
+var shapesParts = map[model.Shape](map[model.Rotation](map[int]string)){
```

Comment:
This one isn't super legible. I understand it but it requires too much effort IMO